src/sider/arbeidsuforhet/RundskrivLenke.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, Heading } from "@navikt/ds-react";
+import React from "react";
+import { EksternLenke } from "@/components/EksternLenke";
+
+const texts = {
+  heading: "Nyttig lenke",
+  link: "Rundskriv på 8-4 arbeidsuførhet",
+};
+
+const rundskrivUrl =
+  "https://lovdata.no/pro/#document/NAV/rundskriv/r08-00?from=NL/lov/1997-02-28-19/%C2%A78-4";
+
+export default function RundskrivLenke() {
+  return (
+    <Box
+      background="surface-default"
+      padding="6"
```

Comment:
Utdraget som ligger over har padding 4, mens de andre boxene på 8-4 siden har padding 6. Vi har nok 6 de fleste steder, selv om jeg personlig liker 4 bedre. Bare lurte på om det så penere ut med 4 her siden boksen rett over har 4?

Reply:
Helt enig! Det har jeg bare ikke spotta, så godt sett! Skulle helst hatt det likt overalt, ihvertfall på samme side. Jeg retter det opp her ihvertfall.

Reply:
Ser at 'utdrag fra sykefraværet' er det eneste elementet på siden som har padding 4. De andre boksene på siden, som feks 'siste vurdering' og 'historikk' har padding 6. Hadde det vært mulig å bare fikset opp i dette en gang for alle? Gjelder egt også spacen rundt boksene som også er like random.

Reply:
Ser at 'utdrag fra sykefraværet' er det eneste elementet på siden som har padding 4. De andre boksene på siden, som feks 'siste vurdering' og 'historikk' har padding 6. Hadde det vært mulig å bare fikset opp i dette en gang for alle? Gjelder egt også spacen rundt boksene som også er like random.